_emails/us/california/santamaria.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Santa Cruz, CA
+permalink: "/santacruz"
+name: Letter to City Council
+city: Santa Maria
+state: CA
+layout: email
+recipients:
+- CityCouncil@cityofsantamaria.org
+- apatino@cityofsantamaria.org
+- gsoto@cityofsantamaria.org
+- ewaterfield@cityofsantamaria.org
+- mmoats@cityofsantamaria.org
+- mcordero@cityofsantamaria.org
+subject: [\*\*\* INSERT UNIQUE SUBJECT LINE \*\*\*]
```

Comment:
even though we're probably doing away with subject lines altogether, no need for escape characters here.